### PR TITLE
DataFrame.cumsum()

### PR DESCRIPTION
**Pull request by RainFung**

### CODE REVIEW

File: databricks/koalas/frame.py [hunk 1]
```diff
@@ -3895,6 +3896,71 @@ def describe(self, percentiles: Optional[List[float]] = None) -> 'DataFrame':
                                   index_map=[('summary', None)])
         return DataFrame(internal).astype('float64')
 
+        # TODO: add 'axis' parameter
```
Comment (HyukjinKwon) [inline databricks/koalas/frame.py]: Seems we can remove leading 4 spaces here

File: databricks/koalas/frame.py [hunk 2]
```diff
@@ -3987,6 +3988,71 @@ def describe(self, percentiles: Optional[List[float]] = None) -> 'DataFrame':
...
+        -------
+        Series or DataFrame
+
+        See Also
+        --------
+        DataFrame.sum : Return the sum over DataFrame axis.
+        DataFrame.cummax : Return cumulative maximum over DataFrame axis.
+        DataFrame.cummin : Return cumulative minimum over DataFrame axis.
+        DataFrame.cumsum : Return cumulative sum over DataFrame axis.
+        DataFrame.cumprod : Return cumulative product over DataFrame axis.
```
Comment (garawalid) [inline databricks/koalas/frame.py]: We don't have yet `DataFrame.cumprod`. Would you mind delete it.

File: databricks/koalas/frame.py [hunk 3]
```diff
@@ -3987,6 +3988,71 @@ def describe(self, percentiles: Optional[List[float]] = None) -> 'DataFrame':
...
+        Returns
+        -------
+        Series or DataFrame
+
+        See Also
+        --------
+        DataFrame.sum : Return the sum over DataFrame axis.
+        DataFrame.cummax : Return cumulative maximum over DataFrame axis.
+        DataFrame.cummin : Return cumulative minimum over DataFrame axis.
+        DataFrame.cumsum : Return cumulative sum over DataFrame axis.
```
Comment (garawalid) [inline databricks/koalas/frame.py]: The same here. You are referring to the same page `DataFrame.cumsum`.

File: databricks/koalas/tests/test_dataframe.py [hunk 4]
```diff
@@ -778,3 +778,9 @@ def get_data():
 
         with self.assertRaises(NotImplementedError):
             left_kdf.update(right_kdf, join='right')
+
+    def test_cumsum(self):
+        pdf = pd.DataFrame([[2.0, 1.0], [3.0, None], [1.0, 0.0]], columns=list('AB'))
```
Comment (garawalid) [inline databricks/koalas/tests/test_dataframe.py]: Please use this test instead. 
```python
        pdf = pd.DataFrame([[2.0, 1.0], [5, None], [1.0, 0.0],[2.0, 4.0],[4.0, 9.0]], columns=list('AB'))
        kdf = ks.from_pandas(pdf)
```

**Comment by HyukjinKwon** [comment]
Im going to take this over at https://github.com/databricks/koalas/pull/521

**Pull request closed without merging**